test/e2e/kind-config.yaml

```diff
@@ -0,0 +1,19 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  ipFamily: dual
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 9080
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 9443
+    protocol: TCP
```

Comment:
GitHub looks happier when a file ends with a linebreak

Reply:
fixed

Reply:
FYI, that comment applies to all the files that end with no linebreak.

Reply:
fixed for all files found with no linebreak at the and.